Gruntfile.js

```diff
@@ -23,8 +23,24 @@ module.exports = function (grunt) {
   var appConfig = {
     app: require('./bower.json').appPath || 'app',
     dist: 'client/dist'
+  }
+    // Configurable paths for the application
+  var bowerJson = require('./bower.json');
+  var bowerRc;
+  try {
+      bowerRc = JSON.parse(fs.readFileSync('./.bowerrc'), 'utf-8');
+  } catch(err) {
+      // No .bowerrc.  Not a problem--it's optional!
+  }
+  var appConfig = {
+      app: (bowerJson && bowerJson.appPath) ? bowerJson.appPath : 'client/ngapp',
+      // vendor: (bowerRc && bowerRc.directory) ? bowerRc.directory : 'client/ngapp/vendor',
+      vendor: (bowerRc && bowerRc.directory) ? bowerRc.directory : 'bower_components',
+      dist: 'client/dist',
+      stage: '.tmp'
```

Comment:
I see good in all of that except the part about moving the appPath to .bowerrc.  Bower's documentation lists all the properties consumed from there and doesn't name appPath at all.  Looking into the Yeoman generator projects also shows that Yeoman has either established or conformed to a precedent for getting the appPath directory from bower.json, so it might be confusing to break from that pattern.

That said, editing the proposal to presume the file will exist means I can drop the ternary operator use and leave the fs.readFileSync() call unprotected by try/catch.  Those two items alone will yield a much smaller footprint.